web3-dev/smart-contracts/getting-started.rst

```diff
@@ -2,41 +2,40 @@
 
 .. _sc-getting-started:
 
-Introduction to Smart Contracts
-===============================
+Getting started
+===============
 
 In this section you will learn how to compile your first Massa smart contract.
 
 Setting up a new project
 ------------------------
 
 Make sure you have a recent version of Node.js and npm. Update or `install <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_ them if needed.
-Create or go to the directory where you want to set up your project and run:
+`massa-sc-toolkit <https://github.com/massalabs/massa-sc-toolkit/>`_ is a tool that creates a boilerplate smart-contract project.
+To create a smart-contract project, invoke the toolkit by running:
 
 .. code-block:: shell
 
-   npm install --global yarn npx
-   npx massa-sc-create my-sc
+  npx @massalabs/sc-toolkit init my-sc
 
 Now that the npm project is created, go inside your smart-contract directory and install the dependencies using the following commands:
 
 .. code-block:: shell
 
    cd my-sc
-   npm install --legacy-peer-deps
+   npm install
```

Comment:
Is that still necessary ? I have the impression that massalabs/massa-sc-toolkit comes with the node_modules. In that case I would simply say to open the folder with your favorite IDE. Or remove this line altogether.

Reply:
you're right, the toolkit performs the `npm install`